package.json

```diff
@@ -46,7 +48,8 @@
     "router5-plugin-browser": "^7.0.1",
     "showdown": "^1.9.0",
     "throttle-debounce": "^2.0.1",
-    "universal-ga": "^1.2.0"
+    "universal-ga": "^1.2.0",
+    "webpack": "^4.28.2"
```

Comment:
Do we need to remove this as a devDependency?

Reply:
nopes, still needed when running locally in dev mode. When we deploy to GCP, prod being the default devDependencies are ignored.